VisualRiders.PointOfSale.Project/Controllers/StaffMembersController.cs

```diff
@@ -1,9 +1,67 @@
 using Microsoft.AspNetCore.Mvc;
+using VisualRiders.PointOfSale.Project.DTOs;
+using VisualRiders.PointOfSale.Project.Models;
+using VisualRiders.PointOfSale.Project.Services;
 
 namespace VisualRiders.PointOfSale.Project.Controllers;
 
 [ApiController]
-[Route("staffMembers")]
+[Route("staff")]
 public class StaffMembersController : ControllerBase
 {
+    private readonly StaffMembersService _service;
+
+    public StaffMembersController(StaffMembersService service)
+    {
+        _service = service;
+    }
+
+    [HttpGet]
+    public List<ReadStaffMemberDto> GetAll()
+    {
+        return _service.GetAll();
+    }
+
+    [HttpPost]
+    public ActionResult<ReadStaffMemberDto> Create(CreateUpdateStaffMemberDto payload)
+    {
+        try
+        {
+            return _service.Create(payload);
+        }
+        catch (Exception ex)
+        {
+            return BadRequest(ex.StackTrace);
```

Comment:
I don't think it's a good idea to show stack trace to the user as this could leak sensitive information about the system but the way you handled the exception is quite nice